service-discovery/api/src/main/java/io/smallrye/discovery/ServiceDiscovery.java

```diff
@@ -0,0 +1,10 @@
+package io.smallrye.discovery;
+
+import java.util.List;
+import java.util.concurrent.CompletionStage;
+
+public interface ServiceDiscovery {
```

Comment:
I thought of this entity as something that would provide `ServiceInstance`s for a specific service name